plugin/src/main/resources/io/jenkins/plugins/casc/assets/prism.css

```diff
@@ -1,5 +1,23 @@
+/* Custom Jenkins bits to provide a good user experience
+ */
+
+iframe,
+.yui-overlay,
+#page-head,
+#menuSelector,
+footer
+ {
+	-webkit-user-select: none;
+	-moz-user-select: none;
+	-ms-user-select: none;
+	user-select: none;
```

Comment:
looks fine, shame about the prefixing:
https://caniuse.com/#feat=user-select-none

Reply:
The docs is a mess :cry: 
https://developer.mozilla.org/en-US/docs/Web/CSS/user-select